### ISSUE TITLE
Online Storyboarder Collaboration

### ISSUE THREAD

**Issue by Corruptinator** [wonderunit/storyboarder#689]
This might be advanced to tackle but what if there was an online storyboarding collaboration within Storyboader? I can imagine that the author/artist can host his or her own domain to share their own storyboard progress with other workers/artist and determine what needs to be changed either by adding or editing new frames, annotations, notes, etc.

Advanced, yes. But an idea nonetheless.

**Comment by deomacius** [comment]
Could something like this be accomplished with Dropbox, Google Drive or iCloud integration? Point to the file in one of those locations (set in user prefs) and let the service manage access rights, etc?

**Comment by Rayek** [comment]
Storyboarder files save in open formats, and it works quite well with Git/Github (I tested). They only drawback for now is that Storyboarder does not automatically update its contents based on changes that occur externally. Which means Storyboarder must be restarted before making changes/after a pull.

Github is a perfect collaboration platform. Use GitKraken to simplify the workflow in your team.

**Comment by audionerd** [comment]
Yes, eventually we'd like for Storyboarder to refresh when the Fountain script changes. Maybe even track filesystem changes to boards? That will make collaboration a bit easier. This will probably be a 2.0 feature. Tracking here: https://github.com/wonderunit/storyboarder/issues/76
